### PR TITLE
added sticky footer adjustment for admin toolbars

### PR DESCRIPTION
**Pull request by rburkey87**
Fixes issue #59

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+jQuery( document ).ready(function( $ ) {
+
+    if (document.querySelector("#toolbar-bar") != null) {
+        $("body").addClass("admin-menu-bar");
+    }
+
+    if (document.querySelector("#toolbar-item-administration-tray") != null) {
```
Comment (katrialesser) [inline js/scripts.js]: make sure - probably still exists even if hidden. may have to see if it's displayed or see if the classes INSIDE that menu change.

**Comment by rburkey87** [comment]
It works great now, but please look at the setTimeout within the function and make sure the loop won't inhibit performance.

File: js/scripts.js [hunk 2]
```diff
@@ -12,9 +12,10 @@ jQuery( document ).ready(function( $ ) {
         }
         else {
             $("body").removeClass("admin-extended-menu-bar");
-            setTimeout(checkForChanges, 500);
         }
-
-        jQuery(checkForChanges);
+        setTimeout(checkForChanges, 500);
```
Comment (katrialesser) [inline js/scripts.js]: Please change this js to match an on-click approach instead.

File: js/scripts.js [hunk 3]
```diff
@@ -4,18 +4,18 @@ jQuery( document ).ready(function( $ ) {
         $("body").addClass("admin-menu-bar");
     }
 
-    function checkForChanges() {
-        var extendedToolbar = jQuery("#toolbar-item-administration-tray");
+    var extendedToolbar = jQuery("#toolbar-item-administration-tray");
```
Comment (katrialesser) [inline js/scripts.js]: it would be more consistent to do $ everywhere instead of jQuery here @rburkey87

File: js/scripts.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+jQuery( document ).ready(function( $ ) {
+
+    if (document.querySelector("#toolbar-bar") != null) {
+        $("body").addClass("admin-menu-bar");
+    }
+
+    var extendedToolbar = jQuery("#toolbar-item-administration-tray");
```
Comment (katrialesser) [inline js/scripts.js]: later we should change jQuery to $ for consistency @rburkey87

**Pull request merged**